doc/Overview/Mvux/Feeds.md

```diff
@@ -0,0 +1,259 @@
+---
+uid: Overview.Mvux.Feeds
+---
+
+# What are Feeds?
+
+Feeds are there to manage asynchronous data requests from a service and provide their result to the View in an efficient manner.
+
+It provides out of the box support for data coming from task-based methods as well Async-Enumerables ones.  
+
+They accompany the requests with additional metadata that tell whether the request is still in progress, ended in an error, and if it was successful, whether the data that was returned contains any entries or was empty.
+
+## Feeds are stateless
+
+Feeds are used as a gateway to request data from services, but it does not hold the data. The data is rather sent over to the requesting party awaiting the feed itself (see [below](#directly-await-feeds)).
+
+As such, feeds do not provide support for reacting upon changes the user makes to the data, and the data is preserved in its original state.  
```

Comment:
Not sure what you mean by this - sounds like feeds won't show the latest data but I don't think that's what you mean

Reply:
I meant it doesn't update itself by the View side changes. Yes the content does need some sharpening.

Reply:
Rephreased and explained. Please review.